extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -27,14 +27,14 @@ class AmpStickyAd extends AMP.BaseElement {
   constructor(element) {
     super(element);
 
+    /** @const @private {!../../../src/service/vsync-impl.Vsync} */
+    this.vsync_ = this.getVsync();
```

Comment:
since `viewport_` was changed to be assigned in `buildCallback`, do the same for `vsync_`?